tools/report-converter/tests/unit/eslint_output_test_files/Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	eslint -f json files/index.js | python -m json.tool > ./reports.json
```

Comment:
Do we run this unit test anywhere? I believe not, because it would require `eslint` installed on that computer. So is this Makefile here for some future feature?

Reply:
No, we do not run this target anywhere but we need it because the user can see from this how to reproduce the reports.json test file. @gyorb required this in my other pull request (https://github.com/Ericsson/codechecker/pull/2535#discussion_r393026203).